src/python/pants/backend/python/dependency_inference/rules.py

```diff
@@ -177,6 +204,25 @@ async def infer_python_dependencies_via_imports(
         if maybe_disambiguated:
             merged_result.add(maybe_disambiguated)
 
+        if not owners.unambiguous and imp.split(".")[0] not in DEFAULT_UNOWNED_DEPENDENCIES:
+            unowned_imports.add(imp)
+
+    unowned_dependency_behavior = python_infer_subsystem.options.unowned_dependency_behavior
+    if unowned_imports and unowned_dependency_behavior is not UnownedDependencyUsage.DoNothing:
+        raise_error = unowned_dependency_behavior is UnownedDependencyUsage.RaiseError
+        log = logger.error if raise_error else logger.warn
+        log(
+            f"The following imports in {address} have no unambiguous owner:\n\n{bullet_list(unowned_imports)}\n\n"
```

Comment:
```suggestion
            f"The following imports in {address} have no owners:\n\n{bullet_list(unowned_imports)}\n\n"
```

Sorry for the pendantry: the distinction is that if something was ambiguous, and you would get an ambiguity warning via: https://github.com/pantsbuild/pants/blob/b5a0c28dd3e8d4bbd9a6af67e7b3ddd47448ae3a/src/python/pants/engine/target.py#L1983-L2016

So "no unambiguous" is different from "none at all".

Reply:
pedantry is welcome :)